fields.yml

```diff
@@ -1423,3 +1423,37 @@
           description: >
             Group the user is a part of. This field can contain a list of groups, if
             necessary.
+    
+    - name: user_agent
+      title: User agent
+      group: 2
+      description: >
+        The user_agent fields normally come from a browser request. They often
+        show up in web service logs coming from the parsed user agent string.
+      type: group
+      fields:
+    
+        - name: original
```

Comment:
@webmat I still think we should turn indexing off here: https://github.com/elastic/ecs/pull/138

Reply:
I don't mind turning it off, actually. But didn't we get a request in Beats for precisely the reverse?

Reply:
Of course people can override that, so not a big issue

Reply:
I think that request was for having it as a `text` field? Having is a keyword would still not make ti searchable. I wonder now if not index if a user can still add a `.text` multi-field which is indexed?

Reply:
I don't think it's possible, no. But if that person wants indexing, they can totally re-enable keyword indexing on the canonical field, and add a multi-field for text indexing.

If ECS makes it non-indexed, it means that no application that will depend on the official ECS spec can ever perform searches in that field. So whatever users do with it (re-enabling indexing) will not conflict. Because ECS will say it should not be indexed, so the applications can't depend on this being a `keyword` or a `text` field.

What I want to avoid at all cost is to make this canonical field `text`. If we do `text` here, it must be a multi-field.